guides/phoenix_mix_tasks.md

```diff
@@ -232,7 +232,7 @@ $ mix phx.server
 Compiling 17 files (.ex)
 
 == Compilation error in file lib/hello_web/controllers/post_controller.ex ==
-** (CompileError) lib/hello_web/controllers/post_controller.ex:22: undefined function post_path/3
+** (CompileError) lib/hello_web/controllers/post_controller.ex:22: undefined function Routes.post_path/3
```

Comment:
Forgot to mention this before, but  these CompileErrors will now be 

 > warning:  function HelloWeb.Router.Helpers.post_path/3 is undefined or private
 >    lib/hello_web/controllers/post_controller.ex:22

Reply:
We should also update the text on line 228 to say:

 Important: If we don't do this, we will see the following warnings in our logs, and our application will error when trying to execute the function.

Reply:
Will do, totally forgot that the alias would show missing funs on the aliased module .